cli/cli.go

```diff
@@ -0,0 +1,77 @@
+package cli
+
+import (
+	. "fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/codegangsta/cli"
+	"github.com/ugorji/go/codec"
+
+	"polydawn.net/repeatr/def"
+	"polydawn.net/repeatr/executor"
+	"polydawn.net/repeatr/executor/null"
+)
+
+func GetApp() *cli.App {
+	app := cli.NewApp()
```

Comment:
Eh... We'll want to be able to invoke many times for integration tests, perhaps. But the various cli `Run()` functions do not appear to mutate the struct, so I'll use `init()` because why not :)